test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/BindingSourceModelBinderTest.cs

```diff
@@ -105,7 +105,7 @@ public TestableBindingSourceModelBinder(BindingSource source)
             protected override Task<ModelBindingResult> BindModelCoreAsync([NotNull] ModelBindingContext bindingContext)
             {
                 WasBindModelCoreCalled = true;
-                return Task.FromResult(new ModelBindingResult(null, bindingContext.ModelName, true));
+                return Task.FromResult(new ModelBindingResult(model: null, key: bindingContext.ModelName, isModelSet: false));
```

Comment:
Not exactly cosmetic. I should have tested `isModelSet: false` with `model: null` as well as `isModelSet: true`. Instead I just switched `isModelSet` here. Will make sure both are tested.